docs/website/docs/dlt-ecosystem/destinations/filesystem.md

```diff
@@ -220,11 +220,63 @@ Current default layout: **{table_name}/{load_id}.{file_id}.{ext}`**
 > 💡 Note that the default layout format has changed from `{schema_name}.{table_name}.{load_id}.{file_id}.{ext}` to `{table_name}/{load_id}.{file_id}.{ext}` in dlt 0.3.12. You can revert to the old layout by setting the old value in your toml file.
 
 
+### Available layout placeholders
+
+#### Standard placeholders
+
+* `schema_name` - the name of schema
+* `table_name` - table name
+* `load_id` - the id of the load package form which the file comes from
+* `file_id` - the id of the file
+* `ext` - a format of the file i.e. jsonl or parquet
+
+#### Date and time placeholders
+> 💡 Keep in mind all values are lowercased.
+
+* Years
+  * `YYYY` - 2024, 2025
+  * `Y` - 2024, 2025
+* Months
+  * `MMMM` - January, February, March
+  * `MMM` - Jan, Feb, Mar
+  * `MM` - 01, 02, 03
+  * `M` - 1, 2, 3
+* Days of month
+  * `DD` - 01, 02
+  * `D` - 1, 2
+* Hours 24h format
+  * `HH` - 00, 01, 02...23
+  * `H` - 0, 1, 2...23
+* Minutes
+  * `mm` - 00, 01, 02...59
+  * `m` - 0, 1, 2...59
+* Days of week
+  * `dddd` - Monday, Tuesday, Wednesday
+  * `ddd` - Mon, Tue, Wed
+  * `dd` - Mo, Tu, We
+  * `d` - 0-6
+* `Q` - quarters 1, 2, 3, 4,
+* `timestamp` - timestamp from pendulum.DateTime
+* `load_package_timestamp` - timestamp from [load package](../../general-usage/destination-tables.md#load-packages-and-load-ids)
+
 You can change the file name format by providing the layout setting for the filesystem destination like so:
 ```toml
 [destination.filesystem]
 layout="{table_name}/{load_id}.{file_id}.{ext}" # current preconfigured naming scheme
-# layout="{schema_name}.{table_name}.{load_id}.{file_id}.{ext}" # naming scheme in dlt 0.3.11 and earlier
+
+# More examples
+# With timestamp
+# layout = "{table_name}/{timestamp}/{load_id}.{file_id}.{ext}"
+
+# With timestamp of the load package
+# layout = "{table_name}/{load_package_timestamp}/{load_id}.{file_id}.{ext}"
```

Comment:
Can I specify the `load_package_timestamp` format? Or do I need to use custom  `placeholder_callback` for this? If so, how can I do it?

Reply:
@VioletM `load_package_timestamp` always comes from current load package so you can't specify it.

Reply:
And what format would it use then? `YYYY-MM-DD HH:MM:SS`?

Reply:
no it is an int timestamp